tutor/src/screens/assignment-review/ux.js

```diff
@@ -65,6 +64,7 @@ export default class AssignmentReviewUX {
   }
 
   @computed get scores() {
+    if (!this.selectedPeriod) { return null; }
```

Comment:
I know this is kind of an undesired pattern, but something in the render cycle is trying to compute this before it should. Would like to track it down in the future.